README.md

```diff
@@ -1,17 +1,49 @@
 <p align="center">
   <a href="https://glide-core.crowdstrike-ux.workers.dev">
     <picture alt="Glide Core Logo">
-      <source srcset="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/light.png?raw=true" media="(prefers-color-scheme: light)" />
-      <source srcset="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/dark.png?raw=true" media="(prefers-color-scheme: dark)" />
-      <img src="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/light.png?raw=true" width="500" />
+      <source 
+        srcset="/.github/logos/light.png?raw=true" 
```

Comment:
Pathing from root and not referring to `main` means we can test logo changes on branches. Should work for `main` too 🤞.